lib/exceptions.dart

```diff
@@ -1,55 +1,67 @@
 part of 'torch_light.dart';
 
 class EnableTorchExistentUserException implements Exception {
-  String message;
+  String? message;
 
-  EnableTorchExistentUserException(this.message);
+  EnableTorchExistentUserException([this.message]);
```

Comment:
Please use a constructor with an optional parameter:
`EnableTorchExistentUserException({this.message});`

Reply:
Oh, you wanted the Named Optional parameters! Alright, I'll do it right now :) 
May I ask if there is a reason behind the decision of using Named Optional Parameters rather than Positional Optional Parameters? Thanks :)

Reply:
Because the input `message` param is null by default, hence you could call `EnableTorchExistentUserException()` or `EnableTorchExistentUserException('some message')`. It seems more idomatic to me and the majority of this kind of params follow this pattern.

Reply:
Yeah, that makes sense! Thanks for the answer :)